codan/org.eclipse.cdt.codan.ui.cxx/.settings/org.eclipse.pde.prefs

```diff
@@ -27,7 +27,7 @@ compilers.p.unknown-class=0
 compilers.p.unknown-element=1
 compilers.p.unknown-identifier=0
 compilers.p.unknown-resource=0
-compilers.p.unresolved-ex-points=0
+compilers.p.unresolved-ex-points=1
```

Comment:
This will need something like:

```sh
        if echo $i | grep 'org.eclipse.cdt.codan.ui.cxx' > /dev/null; then
            # Special case, see comment in org.eclipse.cdt.codan.ui.cxx/plugin.xml
            sed -i \
                '-es@compilers.p.unresolved-ex-points=0@compilers.p.unresolved-ex-points=1@' \
                $d/.settings/org.eclipse.pde.prefs
        fi
```

added to `releng/scripts/do_project_settings.sh` in the PDE section

Reply:
Thanks for the code snippet, @jonahgraham. I will integrate and update the PR.